docs/spec.md

```diff
@@ -2263,23 +2263,58 @@ Computes dot products between slices of `lhs` and slices of `rhs` and produces a
 More formally, `result[result_index] = dot_product`, where:
 
 <!-- markdownlint-disable line-length -->
-* `lhs_result_dimensions = [d for d in axes(lhs) and d not in lhs_batching_dimensions and d not in lhs_contracting_dimensions]`.
+* `If lhs uses non-quantized tensor and rhs uses quantized tensor:`
+
+  ```python
+   for each batch_size in [0, dim(lhs, lhs_batching_dimensions[0])]:
```

Comment:
Why we only doing quantization on `lhs_batching_dimensions[0]`?

If this is a generalization of `tfl.fully_connected`, we should quantize on all batches (i.e. `lhs_batching_dimensions[0] * lhs_batching_dimensions[1] * ... * lhs_batching_dimensions[n - 1]`).

Reply:
Hi @loganchien 
Thanks for explaining in detail (offline) what exactly we need here.

As this change is related to the hybrid op (floating point `lhs` and quantized `rhs`) case, I propose that we hold on this change for a while until we  decide on whether the specification requires a hybrid op.
[ref](https://github.com/openxla/stablehlo/pull/1413#discussion_r1183127043).